perun-base/src/main/java/cz/metacentrum/perun/core/api/Attribute.java

```diff
@@ -58,6 +58,29 @@ public Object getValue() {
 		return value;
 	}
 
+	public String valueAsString() {
```

Comment:
I believe, all of these methods will be automatically called, when RPC serialize object. Not sure, if it will fail or simply duplicate the value in returned JSON object, but if you explicitly need these methods, please add them to annotation ``JsonIgnoreProperties`` for ``AttributeMixin`` (project perun-base, package cz.metacentrum.perun.rpc.serializer).

Reply:
No, these methods are not properties because their names do not start with "get".